firmware_configurations/klipper/jaeger/Spider_1.1_V1.8/readme.md

```diff
@@ -0,0 +1,5 @@
+<B>VORON V1.8 FYSETC Spider Config</B>
+
```

Comment:
Can you add your image to this README?  Something like this

    ![Spider Wiring](Voron_1_Spider_11_Wiring.png)

Reply:
Sure, will do. Thanks for the review!